### PR TITLE
WA-393 Removing paper wallet on success

### PR DESCRIPTION
**Pull request by DmitryBespalov**

### CODE REVIEW

File: MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift [hunk 1]
```diff
@@ -192,6 +192,9 @@ public class WalletApplicationService: Assertable {
             do {
                 try markDeploymentSuccess()
                 try finishDeployment()
+                if let paperWallet = ownerAddress(of: .paperWallet) {
```
Comment (sche) [inline MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift]: how about force unwrapping here?

Comment (sche) [reply MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift]: in createWalletInBlockchain() is seems like we always execute markDeploymentAcceptedByBlockchain. What if createWallet service was not available?

Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift]: force unwrapping - i don't want to, it's forced unwrapping :(

if service is not available, then completion will return error. let me see if we can throw earlier

Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift]: yes, the throw is better

File: MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift [hunk 2]
```diff
@@ -183,7 +183,7 @@ public class WalletApplicationService: Assertable {
             throw Error.missingWalletAddress
         }
         let service = DomainRegistry.blockchainService
-        service.createWallet(address: address.value, completion: didFinishDeployment(success:error:))
+        try service.createWallet(address: address.value, completion: didFinishDeployment(success:error:))
```
Comment (sche) [inline MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift:186]: In general this method is implemented as sync. So it is cleat why we call next line. But it has a completion that makes me think that it is async. => Confusion :)

**Review by sche — APPROVED** [review]

**Pull request merged**